util/package.go

```diff
@@ -22,9 +22,34 @@ import (
 const defaultType = "integration"
 
 var CategoryTitles = map[string]string{
-	"logs":     "Logs",
-	"metrics":  "Metrics",
-	"security": "Security",
+	"aws":               "AWS",
+	"azure":             "Azure",
+	"cloud":             "Cloud",
+	"config_management": "Config management",
+	"containers":        "Containers",
+	"crm":               "CRM",
+	"Custom":            "Custom",
```

Comment:
all the others have a lowercase key name

Reply:
good catch, fixed.